metrics/storage/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Set up an Ubuntu image with 'blogbench' installed
+
+# Usage: FROM [image name]
+FROM ubuntu
+
+RUN apt-get update && \
+	apt-get install -y build-essential wget && \
+	wget https://download.pureftpd.org/pub/blogbench/blogbench-1.1.tar.gz && \
+	tar xvzf blogbench-1.1.tar.gz && \
+	cd blogbench-1.1 && \
+	./configure && \
+	make && \
+	make install-strip
+
```

Comment:
I just noticed yesterday that, if you are editing this file, we could also delete the .tar.gz file from the image (or untar it in place) now the binary is installed. and/or the same for the source tree once the binary is in place. Just a thought.